<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupEventItem</class>
 <widget class="QWidget" name="GroupEventItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupEventItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-family:&quot;Microsoft YaHei&quot;;
	
	background-color: rgb(255, 255, 255);
}
#label
{
	border:1px solid  rgb(0, 0, 0);
}

QToolButton
{
	border:1px solid  gray;
	border-radius:4px;
	max-height:22px;
	width:40px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="JChat::Label" name="label">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请求添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMessage">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>附加消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>168</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>已同意</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonPass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>同意</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonReject">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>拒绝</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>JChat::Label</class>
   <extends>QLabel</extends>
   <header>Label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
